.github/workflows/conflicts.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Merge Conflict Watcher'
+on:
+  pull_request:
+    types: [synchronize, edited]
```

Comment:
Shouldn't have removed `opened`

Final change

Reply:
> Shouldn't have removed `opened`

Oops, sorry. I've added it now :)